crates/pyhq/python/hyperqueue/task/task.py

```diff
@@ -1,12 +1,21 @@
-from typing import Dict, Sequence
+from typing import Optional, Sequence
+
+from hyperqueue.ffi.protocol import ResourceRequest
 
 TaskId = int
 
 
 class Task:
-    def __init__(self, dependencies: Sequence["Task"] = ()):
+    def __init__(
+        self,
+        task_id: int,
```

Comment:
`int` -> `TaskId`

Reply:
There is `TaskId` on line 5 of this file, right? You also added another here: [ffi/connection.py](https://github.com/It4innovations/hyperqueue/pull/389/files/674a93bda9f2191f12cb0859c2f83a46fea8642b#diff-163d04ce001322a9df03dd530d5e3bfefadc19b4e03c6e1a4055266007e4b19b).

Reply:
Oh, I was looking in a wrong branch